packages/mobile/src/components/CountdownCircleTimer.jsx

```diff
@@ -70,7 +71,7 @@ const CountdownCircleTimer = (props) => {
         )}
         <Path
           fill="none"
-          strokeWidth={strokeWidth}
+          strokeWidth={trailStrokeWidth || strokeWidth}
```

Comment:
I would suggest here to check if the value is set, a.k.a. it is not undefined. Because in this case if we set `trailStrokeWidth` to 0 it will use the `strokeWidth` value when we want that path to be 0. So something like this: `strokeWidth={trailStrokeWidth !== undefined ? trailStrokeWidth : strokeWidth}`

Reply:
Great point I totally missed that edge case. I have currently replaced `OR` operator with `Nullish coalescing operator (??)`.